its/it/env.ts

```diff
@@ -0,0 +1,21 @@
+import { GITHUB_MAIN_BRANCH } from "../constant";
+
+export function loadEnvironmentVariables() {
+  const { SONARCLOUD_TOKEN, AZURE_TOKEN } = process.env;
```

Comment:
I agree. Since there's a bit more to do (all other variables could be harmonized as well) I've created [SONARAZDO-433](https://sonarsource.atlassian.net/browse/SONARAZDO-433) to take care of it after this PR is merged, if that sounds good